android/Staccato_AN/app/src/main/java/com/on/staccato/data/UserInfoPreferencesManager.kt

```diff
@@ -21,8 +47,36 @@ class UserInfoPreferencesManager(context: Context) {
         }
     }
 
+    suspend fun setMemberProfile(memberProfile: MemberProfile) {
+        setProfileImageUrl(memberProfile.profileImageUrl ?: "")
+        setNickname(memberProfile.nickname)
+        setRecoveryCode(memberProfile.uuidCode)
+    }
+
+    suspend fun setProfileImageUrl(url: String) {
```

Comment:
url은 null일 수 있으니 파라미터 타입을 nullable로 설정하면 더 명시적일 것 같아요!

```suggestion
    suspend fun setProfileImageUrl(url: String?) {
```